src/Components/Cards/Categories.jsx

```diff
@@ -0,0 +1,40 @@
+
+import "../../styles/Homepage.css";
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+
+const Categories = () => {
+
+    const [categories, setCategories] = useState([]);
+
+      async  function getCategories  () {
+      try {
+        const res = await axios.get("/api/categories");
+        console.log(res.data)
+        setCategories(res.data.categories);
+      } catch (error) {
+        console.log(error);
+      }
+    }
```

Comment:
You can abstract these functions out in a different folder like Category-utils.

Reply:
will surely try it this way ,thanx